src/cli/onefuzz/api.py

```diff
@@ -557,21 +557,51 @@ def get_files(
         report_bytes = self.onefuzz.containers.files.get(report_container, report_name)
         report = json.loads(report_bytes)
 
+        crash_info = {
+            "input_blob_container": None,
+            "input_blob_name": None,
+            "job_id": None,
+        }
+        if "input_blob" in report:
+            crash_info["input_blob_container"] = report["input_blob"]["container"]
+            crash_info["input_blob_name"] = report["input_blob"]["name"]
+            crash_info["job_id"] = report["job_id"]
+        elif "original_crash_test_result" in report:
```

Comment:
`original_crash_test_result` is optional while `crash_test_result` is required for a `RegressionReport`, see the model here:
https://github.com/microsoft/onefuzz/blob/62c3f079af465316f1f6e360142774847918ad29/src/ApiService/ApiService/OneFuzzTypes/Model.cs#L544-L548

But, since we need the input blob from the original report, we should just check that both crash_test_result and original_crash_test_result are present before continuing and reporting an error otherwise.

Ex:

```python
elif "original_crash_test_result" in report and "crash_test_result" in report:
    # assume both are present
else:
    # log error
```